### DISCUSSION TITLE
Getting "JWSError JWSInvalidSignature" "PGRST301"

### DISCUSSION THREAD

**Discussion by kirkstrobeck** [supabase/supabase#18077] [Questions] [answered]
I’m getting this error message after going thru all of these steps https://supabase.com/docs/guides/auth/auth-helpers/nextjs

I’m using `supabase start` to run the local development Docker

The only results I saw were related to self-hosting and mention something about [changing the JWT](https://github.com/PostgREST/postgrest/issues/977#issuecomment-329917367), which I don’t think applies in this case

```sh
pnpm supabase --version
1.100.1

node --version
v18.12.1

pnpm next --version
Next.js v13.4.19
```

**Comment by GabriPalmyro** [comment]
Same error here, but using realtime database

**Comment by kirkstrobeck** [comment, marked as answer]
My `service_role key` was an incorrect value in my env file

When you run `supabase start`, you will see the anon key and service key, those need to be correctly set in your env file
